sqlglot/optimizer/qualify_columns.py

```diff
@@ -154,14 +154,16 @@ def _qualify_columns(scope, schema):
 
             column_table = unambiguous_columns.get(column_name)
 
-            if not scope.is_subquery:
+            if not scope.is_subquery and scope.scope_type != ScopeType.UNNEST:
```

Comment:
`is_subquery` just checks the scope type. For consistency, maybe we should have a `is_unnest` property?

Reply:
done